src/ReadLog.jl

```diff
@@ -0,0 +1,134 @@
+# Read LCM log files directly
+
+# export
+#   LCMlog,
+#   close,
+#   readNextEvent,
+#   handle,
+#   subscribe
+
+# also see JuliaLang's RawFD type, but little information found.
+# www.c4learn/c-programming/c-file-structure-and-file-pointer
+mutable struct FILE
+  level::Cshort
+  token::Cshort
+  bsize::Cshort
+  fd::Cuchar
+  flags::Cuint  # (unsigned flags ;)
+  hold::Cuchar
+  buffer::Ptr{Cuchar}
+  curp::Ptr{Cuchar}
+  istemp::Cuint # (unsigned is temp)
+end
+
+mutable struct lcm_eventlog_t
+  f::Ptr{FILE}
+  eventcount::Clonglong
+end
+const _LCMlog = Ptr{lcm_eventlog_t}
```

Comment:
Can we delete this alias? It's only used in a couple of places, and at least one is marked with a comment indicating what the alias stands for. `Ptr{lcm_eventlog_t}` is pretty short and quite clear.

Reply:
Sure, I was trying something else and just left it there in the end. Will swap it out.

Reply:
Thanks!